guide-app/src/Nav.js

```diff
@@ -114,7 +120,7 @@ export default class Nav extends Component<Props> {
       <ViewContainer>
         <StatusBar
           translucent={ios}
-          barStyle="light-content"
+          barStyle="dark-content" // @TODO `barStyle` should probably be added to `Colors`
```

Comment:
On the home screen the bar is purple on Android. Tested adding a `<StatusBar barStyle="dark-content" backgroundColor={Colors.white} />`to `HomeScreen` but then map is white instead.. so we should look at that

Reply:
Sounds good, I'll look into it a bit more